fr-fr/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
+Timestamp=2017,10,6,22,19,4
+Version=4
+ViewMode=2
```

Comment:
What is this file? I don't think this file is present in original repo!

Reply:
Sorry, this is a manjaro system file. I forgot to put it in .gitignore. It's done!